christfellowshipchurch/web-app-v2
# Update Connect Card

### About
This PR adds the `other` field to our Connect Card form on the website.

### Test Instructions
* fill out form and write text in the `Other` field
* If successful see the changes in Rock [here](https://rock.gocf.org/page/288?WorkflowTypeId=902).

### Screenshots
<img width="1358" alt="image" src="https://github.com/christfellowshipchurch/web-app-v2/assets/46049974/88e8be78-4722-4d95-9423-206a8a99412a">

### Closes Tickets
[CFDP-2064](https://christfellowshipchurch.atlassian.net/browse/CFDP-2064)

[CFDP-2064]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2064?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
